ui/src/wishList/wish/index.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import './styles.scss'
+import kidImg from '../../assets/images/icn_To_Be_Astronaut_130x130.png'
+import sponsorImg from '../../assets/images/icn_To_Be_Astronaut_130x130.png'
+import ImgPlaceholder from '../imgPlaceholder'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faChevronRight } from '@fortawesome/free-solid-svg-icons'
+
+const Wish = ({ wish }) => {
+  const { child, details } = wish
+
+  return (
+    <li className="wish">
+      <div className="date">
+      </div>
+      {false ? <img src={kidImg} alt="child" /> : <ImgPlaceholder text="Add Image" />}
```

Comment:
Seems like this will never show the actual image. Is this intentional until we have that feature?

Reply:
yea, that's correct

Reply:
will add placeholder for image flag in the state for now